src/libstd/num/f64.rs

```diff
@@ -628,6 +628,58 @@ impl f64 {
         unsafe { cmath::nextafter(self, other) }
     }
 
+    /// Returns floating point exponent of the input
+    ///
+    /// ```
+    /// use std::f64;
+    ///
+    /// let x = 0.5f64;
+    /// let abs_diff = (x.logb() - 1).abs();
+    ///
+    /// assert!(abs_diff <= f64::EPSILON)
+    /// ```
+    #[unstable(feature = "std_misc",
+                reason = "unsure about its place in the world")]
+    #[inline]
+    pub fn logb(self) -> f64 {
+        unsafe { cmath::logb(self) }
+    }
+
+    /// Returns integer exponent of the input
+    ///
+    /// ```
+    /// use std::f64;
+    ///
+    /// let x = 0.5f64;
+    /// let abs_diff = (x.ilogb() - 1.0f64).abs();
+    ///
+    /// assert!(abs_diff <= f64::EPSILON)
+    /// ```
+    #[unstable(feature = "std_misc",
+                reason = "unsure about its place in the world")]
+    #[inline]
+    pub fn ilogb(self) -> i64 {
+        unsafe { cmath::ilogb(self) }
+    }
+
+    /// Returns the value of the gamma function
+    /// at the given point
+    /// ```
+    /// use std::f64;
+    ///
+    /// let x = 1.2f64;
+    /// let g = 0.9182f64;
+    /// let abs_diff = (x.gamma() -f ).abs();
+    ///
+    /// assert!(abs_diff <= f64::EPSIOLON)
```

Comment:
I think that EPSILON here may be misspelled, and you can also be sure to run these tests via:

```
make check-stage1-doc-crate-std
```